srcpkgs/singularity/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'singularity'
+pkgname=singularity
+version=3.7.1
+revision=1
+wrksrc="${pkgname}"
+build_style=go
+go_import_path=github.com/sylabs.io/singularity
+go_mod_mode=vendor
+hostmakedepends="wget pkg-config go cryptsetup"
+makedepends="libressl-devel libseccomp-devel"
+depends="squashfs-tools cryptsetup"
+short_desc="HPC centric container platform"
+maintainer="Olaf Mersmann <olafm@p-value.net>"
+license="BSD-3-Clause-LBNL"
+homepage="https://sylabs.io/singularity/"
+distfiles="https://github.com/hpcng/${pkgname}/releases/download/v${version}/${pkgname}-${version}.tar.gz"
+checksum=82d2c65063560195ec34551931be3c325b95e8e2009e92755fd7daad346e083c
+nocross=yes
```

Comment:
What kind of errors did you get?

Reply:
The `makeit` configure doesn't support GO crossbuilds so it chokes wenn GOARCH is set:
```
...
 checking: host C++ compiler... c++
 checking: host Go compiler (at least version 1.13)... not found!
mconfig: could not complete configuration
```

I'd really like to avoid patching crossbuild support into their Makefiles.

Reply:
Heh, np. You could consider making changes upstream, though :P